nginx.conf

```
@@ -0,0 +1,13 @@
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+    root /usr/share/nginx/html;
+    index index.html index.;
+    server_name _;
+
+    # Enable gzip
+    gzip on;
+    gzip_proxied any;
+    gzip_types text/plain text/xml text/css application/x-javascript;
```

```suggestion
    gzip_types text/plain text/xml text/css application/javascript;
```
is x-javascript used?

---

I don't think so.